editor/src/components/inspector/sections/component-section/data-picker-popup.tsx

```diff
@@ -116,20 +116,17 @@ function isChildrenProp(path: PropertyPath): boolean {
 export interface DataPickerPopupProps {
   closePopup: () => void
   style: React.CSSProperties
+  targetPath: ElementPath | null // TODO make it not nullable
   propPath: PropertyPath
   propExpressionPath: Array<string | number> | null
 }
 
 export const DataPickerPopup = React.memo(
   React.forwardRef<HTMLDivElement, DataPickerPopupProps>((props, forwardedRef) => {
-    const { closePopup, propPath, propExpressionPath } = props
+    const { closePopup, propPath, propExpressionPath, targetPath } = props
 
     const [preferredAllState, setPreferredAllState] = useAtom(DataPickerPreferredAllAtom)
 
```

Comment:
there's no funcitonal changes in data-picker-popup, I just changed it so we can feed it with the target path instead of it acquiring its own target path based on selectedViews